src/na/na.c

```diff
@@ -260,20 +263,61 @@ na_finalize(void) NA_DESTRUCTOR;
 
 /*---------------------------------------------------------------------------*/
 #ifdef NA_HAS_DYNAMIC_PLUGINS
+#    ifdef _WIN32
+#        define resolve_plugin_path(offset) NULL
+#    else
+static char *
+resolve_plugin_path(const char *offset)
+{
+    static int placeholder;
```

Comment:
pedantic didn't like using function pointer and casting to void *.  Maybe shutting up the warning is better but this also seems to work

Reply:
ok yes I don't see the warning anymore, thanks